common/djangoapps/third_party_auth/models.py

```diff
@@ -473,8 +467,6 @@ class SAMLConfiguration(ConfigurationModel):
     )
     site = models.ForeignKey(
         Site,
-        null=True,
```

Comment:
@bdero If I recall right `null` in site column was used to mark providers enabled for all sites. How would it work with this version? I mean, is "global" provider configs no longer supported?